#40 menú de pausa

j23rl07 opened the pull request.

j23rl07 committed:
#40-feat: menú de pausa y audio

implementación de audios de prueba y menú de pausa/audio

j23rl07 committed:
feat: menus Básicos

implementados menú de pausa, menú principal y menú de opciones de sonido.
sonidos de prueba instanciados

j23rl07 committed:
fix: localización de archivo cambiada

j23rl07 committed:
fix: datos no guardados

j23rl07 committed:
fix: error en menu principal

ya no se quedan pilladas las animaciones

H3G0A reviewed (changes requested):
- Arreglar el bug de los buffer de los inputs en el menú de pausa.
- Añadir en el menú de opciones un botón para regresar al juego.
- Añadir opciones de resolución.

j23rl07 committed:
feat: retoques del menu

solucionado bug de interacción en menú
añadido opción pantalla completa
añadido textos identificativos menú de audio
etc...

j23rl07 committed:
Merge branch 'develop' into #40-Menú-de-pausa

j23rl07 committed:
resolved conflicts

j23rl07 committed:
#40 feat: añadido pantalla completa al menú principal

H3G0A reviewed (approved):
-Arreglado el problema de inputs fantasma
-Añadido botón de volver al juego
-Añadido toggle para pantalla completa
Se ha acordado dejar la configuración de resolución para más adelante si hay tiempo.
